src/components/mainMenu.js

```diff
@@ -0,0 +1,21 @@
+import MainMenuButton from "./MainMenuButton";
+
+const MainMenu = (onButtonClick) => {
+    const section = document.createElement('div');
+    section.classList.add('mainMenu');
+
+    const peopleButton = MainMenuButton('mainMenu__button','Characters',onButtonClick)
+    section.appendChild(peopleButton)
+
+    const quotesButton = MainMenuButton('mainMenu__button','Quotes',onButtonClick)
+    section.appendChild(quotesButton)
+
+    const deathsButton = MainMenuButton('mainMenu__button','Deaths',onButtonClick)
+    section.appendChild(deathsButton)
```

Comment:
Teraz zauważyłem, że wszystkie mają dodawaną tą samą klasę. To może po prostu lepiej wywalić to z parametrów/argumentów funkcji i po prostu w komponencie MainMenuButton dodawać klasę mainMenu__button? 
```suggestion
    const peopleButton = MainMenuButton('Characters',onButtonClick)
    section.appendChild(peopleButton)

    const quotesButton = MainMenuButton('Quotes',onButtonClick)
    section.appendChild(quotesButton)

    const deathsButton = MainMenuButton('Deaths',onButtonClick)
    section.appendChild(deathsButton)
```

Reply:
Oki, celna uwaga :)